packages/@aws-cdk/aws-cloudwatch/test/test.alarm.ts

```diff
@@ -105,6 +105,7 @@ export = {
       MetricName: 'Metric',
       Namespace: 'CDK/Test',
       Period: 300,
+      Statistic: ABSENT,
```

Comment:
Rather than modify an existing test, could we add a unit test focused on this particular issue - 

```ts
'only one of Statistic or ExtendedStatistic are present in the Alarm'(test:Test) {
}
```

Reply:
Disagree. The test should always have been this, but at the time it was written we didn't have the capacity to express `ABSENT` yet.

Doesn't make sense to me to have 2 tests for the same behavior, that a percentile statistic leads to `ExtendedStatistic` and ONLY `ExtendedStatistic` being set.

Reply:
Is there another test somewhere that verifies `Statistic` property is valid and that `ExtendedStatistic` is `ABSENT`?

Reply:
There is now.